scripts/src/main/resources/scripts/command/terraform

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# Check if terraform is installed.
+function doIsTerraformInstalled() {
+  if command -v terraform &> /dev/null
+  then 
+    return
+  else
+    return 255
+  fi
+}
+
+# Call Terraform with specified arguments.
+function doRun() {
+  doSetup silent
+  terraform "${@}"
+}
+
+function doSetup() {
+
+  if doIsTerraformInstalled 
```

Comment:
Is there are real point in creating this function instead of just calling `command -v terraform &> /dev/null` directly in this `if` statement?
The function `doIsTerraformInstalled` is not called from any where else...

Reply:
No, you are right. It makes no sence. The same can be found in Commandlet docker with doIsDockerInstalled. It is really difficult, if I implement something in different way I was asked why do I implement it not in the same way as usually. For example: I prefer to add an single empty echo after an output to make sure it is visible that outputs generated from different commands. But thats not usual in our environment so I killed that echo again. 

I will change it, but please, tell me where are the developer guides, so I don't fall into the trap again.

By the way: since the last push the commandlet was deleted from my workspace and I don't understand why